centreon/config/packages/monolog.yaml

```diff
@@ -1,19 +1,34 @@
 # Monolog Bundle configuration
 # https://github.com/symfony/monolog-bundle/blob/master/DependencyInjection/Configuration.php#L25
-monolog:
-    channels: ['centreon', 'deprecation']
+when@prod:
+  monolog:
+    channels: [ 'centreon']
     handlers:
-        system:
-            type: error_log
-            level: warning
-            formatter: monolog.error.formater
-            channels: ['!centreon']
-        centreon:
-            type: service
-            id: monolog.debug.handler
-            channels: ['centreon']
-        deprecation:
-            type: rotating_file
-            path: "%log_path%/%kernel.environment%.deprecations.log"
-            max_files: 2
-            channels: [ deprecation ]
+      system:
+        type: error_log
+        level: warning
+        formatter: monolog.error.formater
+        channels: [ '!centreon' ]
+      centreon:
+        type: service
+        id: monolog.debug.handler
+        channels: [ 'centreon' ]
+when@dev:
+  monolog:
+    channels: [ 'centreon', 'deprecation' ]
+    handlers:
+      system:
+        type: error_log
+        level: warning
+        formatter: monolog.error.formater
+        channels: [ '!centreon' ]
+      centreon:
+        type: service
+        id: monolog.debug.handler
+        channels: [ 'centreon' ]
+      deprecation:
+        type: rotating_file
+        path: "%log_path%/%kernel.environment%.deprecations.log"
+        max_files: 2
+        channels: [ deprecation ]
+        file_permission: 662
```

Comment:
IMHO, `664` or `660` should be more appropriate

Reply:
@kduret Not possible because the user used by Gorgoned to run the backup script is not in the group.